compiler-plugin-tests/src/test/resources/test-src/package_14/grpc_unary_pb.bal

```diff
@@ -0,0 +1,170 @@
+        string message;
+        if (req is ContextString) {
+            message = req.content;
+            headers = req.headers;
+        } else {
+            message = req;
+        }
+        var payload = check self.grpcClient->executeSimpleRPC("grpcservices.HelloBallerina/helloBallerina", message, headers);
+        [anydata, map<string|string[]>] [result, respHeaders] = payload;
+        return {content: result.toString(), headers: respHeaders};
+    }
+}
+
+public client class HelloWorldStringCaller {
+    private grpc:Caller caller;
+
+    public isolated function init(grpc:Caller caller) {
+        self.caller = caller;
+    }
+
+    public isolated function getId() returns int {
+        return self.caller.getId();
+    }
+
+    isolated remote function sendString(string response) returns grpc:Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendContextString(ContextString response) returns grpc:Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendError(grpc:Error response) returns grpc:Error? {
+        return self.caller->sendError(response);
+    }
+
+    isolated remote function complete() returns grpc:Error? {
+        return self.caller->complete();
+    }
+
+    public isolated function isCancelled() returns boolean {
+        return self.caller.isCancelled();
+    }
+}
+
+public client class HelloBallerinaStringCaller {
+    private grpc:Caller caller;
+
+    public isolated function init(grpc:Caller caller) {
+        self.caller = caller;
+    }
+
+    public isolated function getId() returns int {
+        return self.caller.getId();
+    }
+
+    isolated remote function sendString(string response) returns grpc:Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendContextString(ContextString response) returns grpc:Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendError(grpc:Error response) returns grpc:Error? {
+        return self.caller->sendError(response);
+    }
+
+    isolated remote function complete() returns grpc:Error? {
+        return self.caller->complete();
+    }
+
+    public isolated function isCancelled() returns boolean {
+        return self.caller.isCancelled();
+    }
+}
+
+public type ContextString record {|
+    string content;
+    map<string|string[]> headers;
+|};
+
+const string ROOT_DESCRIPTOR = "0A10677270635F756E6172792E70726F746F120C6772706373657276696365731A1E676F6F676C652F70726F746F6275662F77726170706572732E70726F746F32560A0A48656C6C6F576F726C6412480A0A68656C6C6F576F726C64121C2E676F6F676C652E70726F746F6275662E537472696E6756616C75651A1C2E676F6F676C652E70726F746F6275662E537472696E6756616C7565325E0A0E48656C6C6F42616C6C6572696E61124C0A0E68656C6C6F42616C6C6572696E61121C2E676F6F676C652E70726F746F6275662E537472696E6756616C75651A1C2E676F6F676C652E70726F746F6275662E537472696E6756616C7565620670726F746F33";
```

Comment:
Name of the `ROOT_DESCRIPTOR ` should change with the latest releases, right?

Reply:
I used previous tool to generate the pb. Will change it

Reply:
Changed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/426/commits/d5024f509bb0866a36bdb6ecc493096076fb4172